spec/acceptance/curb/curb_spec.rb

```diff
@@ -411,6 +411,16 @@
       it_should_behave_like "Curb"
       include CurbSpecHelper::NamedHttp
 
+      it "should reset @webmock_method after each call" do
+        stub_request(:post, "www.example.com").with(body: "01234")
+        c = Curl::Easy.new
+        c.url = "http://www.example.com"
+        c.post_body = "01234"
+        c.http_post
+        expect(c.response_code).to eq(200)
+        expect(c.webmock_method).to eq(:get)
```

Comment:
I don't think we should test internals. We can simply do perform and validate that it was a get request.

Reply:
Yeah you're right. I updated the code @bblimke.